app/TestTCP/testclose.cpp

```diff
@@ -150,6 +149,9 @@ TEST_F(TestEnv_Any, TestClose_Connect_CloseFirst)
 	std::unordered_map<std::string, std::string> accept_env;
 	std::unordered_map<std::string, std::string> connect_env;
 
+    if(char *seed = getenv("RANDOM_SEED"))
+        printf("[RANDOM_SEED : %s]\n", seed);
+
```

Comment:
`E_main` is an emulated main function thus you should refer `env` instead of `getenv`.

Reply:
I think this line is should be kept. Because this code is in `TEST_F`, not `E_Main`, thus it's not bad to use `getenv` in my think. Also, when after applying a suggestion of `accept_env` and `connect_env`, there might be a difference between given `RANDOM_SEED` (which is using for `srand`) and actual `env["RANDOM_SEED"]` which was generated from rand() function.